Makefile

```diff
@@ -51,6 +56,7 @@ help:
 	@echo '  MINIKUBE_DRIVER         Which vm driver to use for Minikube? [default=virtualbox]'
 	@echo '  MINIKUBE_MEMORY         How much memory to allocate for Minikube? [default=3072]'
 	@echo '  MINIKUBE_KUBERNETES     Which Kubernetes version to use with Minikube? [default=v1.16.3]'
+	@echo '  PUBLISH_IMAGES          Should the production images be pushed to the registry? [0|1, default=0]'
```

Comment:
I think `PUBLISH_IMAGES` should be dropped, it is better to do this manually, as presented in Example 9.

Reply:
Yes, it is actually a left over, thanks!